### ISSUE TITLE
a3_custom

### ISSUE THREAD

**Issue by RAW5niper** [BetterDeadThanZed/NamalskAddons#1]
Did you create a3_custom? I use it, but am having an issue I can't resolve. I'm using it to create a trader in Exile Chernarus. Works like a charm (for objects, props, etc). But, on some restarts some of the objects are missing. So, sometimes they're there, sometimes they're not. Driving me nuts.

**Comment by BetterDeadThanZed** [comment]
There have been issues with that where objects go into the ground, or
sometimes into the ground. I don't know of any fixes for it.

**Comment by RAW5niper** [comment]
No, I guess I didn't explain it right. Some of the objects don't appear (right away).
Eventually, you see the missing items pop up. Like they hadn't loaded yet or something.

**Comment by BetterDeadThanZed** [comment]
Sorry, I've never seen that. The only thing I can think of is your server
is taking a while to load everything.
